server/controllers/reviewController/reviewController.js

```diff
@@ -0,0 +1,48 @@
+const router = require('express').Router();
+const ReviewRepository = require('./reviewRepository');
```

Comment:
This is fancy, is this a pattern you learned somewhere? I like it for large projects.

Reply:
The router was what I learned when first getting into node, I think it was new with Express 4. The repository pattern was something I learned when looking into strict OOP languages, which I have also seen adopted in many TypeScript projects. I personally found it helpful to separate the concerns of the controller and direct communication with the database.